### PR TITLE
feat(etl form): improve ETL form

### PR DESCRIPTION
**Pull request by evinracher**
Improve ETL UI desing and style
- Add material ui icons
- Add collapsible components
- Add loader
- Add error messages to inputs

<details>
  <summary>New ETL form</summary>
image
</details>

<details>
<summary>
Error message in inputs
</summary>
image
</detauls

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# IDE
+./idea
```
Comment (nietoga) [inline .gitignore:38]: Why?

**Review by nietoga — APPROVED** [review]
I noticed you removed the `<pre>` tagfrom the logs. That one is required, since we wan't to show the data however the user prints it.

**Comment by evinracher** [comment]
> I noticed you removed the `\<pre\>` tagfrom the logs. That one is required, since we wan't to show the data however the user prints it.

Resolved

File: .gitignore (hunk 1 above)
Comment (evinracher) [reply .gitignore]: Reverted. Please fix the issue with IDE config. It is changing my formatter

**Pull request merged**